### ISSUE TITLE
Support for Multiple Azure DNS zone in App Routing add-on

### ISSUE THREAD

**Issue by ValeruS** [hashicorp/terraform-provider-azurerm#23801]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment and review the [contribution guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/README.md) to help.

### Description

Please add support for multiple Azure DNS zone in App Routing add-on https://github.com/Azure/AKS/issues/3794

From the [documentation](https://learn.microsoft.com/en-us/azure/aks/app-routing?tabs=default%2Cdeploy-app-default#limitations) the application routing add-on supports up to five Azure DNS zones

### New or Affected Resource(s)/Data Source(s)

azurerm_kubernetes_cluster

### Potential Terraform Configuration

```hcl
A comma separated list of resource IDs of the DNS zone resource to use with the web_application_routing addon
```

### References

https://github.com/Azure/AKS/issues/3794

**Comment by rcskosir** [comment]
Thank you for taking the time to open this feature request!

**Comment by stephybun** [comment]
Closed by #26117
